README.md

```diff
@@ -0,0 +1,15 @@
+Hatch Previewer
+===============
+
+This is an app for viewing packaged website data ('hatches') created by [libingester](https://endlessm.github.io/libingester/).
+
+### Usage ###
+
+- From this repo
+ - `npm start -- <path>`
```

Comment:
this should probably use the runner script (`./bin/hatch-preview`) instead of npm start